change/@fluentui-react-divider-b72313a8-0cf1-43ac-b281-3c7f00b83a90.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Added upgrade guide for divider",
```

We should add the correct label

```suggestion
  "comment": "docs: Added upgrade guide for divider.",
```